extensions/amp-auto-ads/0.1/adsense-network-config.js

```diff
@@ -46,13 +47,16 @@ export class AdSenseNetworkConfig {
   getConfigUrl() {
     const docInfo = Services.documentInfoForDoc(this.autoAmpAdsElement_);
     const canonicalHostname = parseUrlDeprecated(docInfo.canonicalUrl).hostname;
+    const win = toWin(this.autoAmpAdsElement_.ownerDocument.defaultView);
     return buildUrl(
       '//pagead2.googlesyndication.com/getconfig/ama',
       {
         'client': this.autoAmpAdsElement_.getAttribute('data-ad-client'),
         'plah': canonicalHostname,
         'ama_t': 'amp',
         'url': docInfo.canonicalUrl,
+        'debug_experiment_id':
```

Comment:
Do we really want to always send this? Might be better to only include it if the hash exists?

Reply:
That is the current state as buildUrl will not include parameter keys that have a null value (which is the case here if the regexp misses given the "|| null" end of the line).  Pardon my code golf.